src/logger/cloudwatch.logger.ts

```diff
@@ -0,0 +1,91 @@
+import AWS from "aws-sdk"
+import Bluebird from "bluebird"
+import winston from "winston"
+import WinstonCloudwatch from "winston-cloudwatch"
+
+import { consoleLogger } from "./console.logger"
+
+// AWS
+const AWS_REGION_NAME = "ap-southeast-1"
+AWS.config.update({ region: AWS_REGION_NAME })
+const awsMetadata = new AWS.MetadataService()
+const metadataRequest = Bluebird.promisify<string, string>(
+  awsMetadata.request.bind(awsMetadata)
+)
+
+// Constants
+// TODO: Check this env var as it is not in example
+const LOG_GROUP_NAME = `${process.env.AWS_BACKEND_EB_ENV_NAME}/nodejs.log`
```

Comment:
Hey I dont understand this comment :( 
this is in reference so some example in the documentation ah?

Reply:
i actually dk too - i tried asking @alexanderleegs but he didn't know too HAHAHAHA

Reply:
for context, this was in the old `logger.js` and i copied this over because it seemed important

Reply:
Oh this comment is because me and Alex tried to find where this log exists, but we could never find it.

Might need to look again and if we conclude this is not being used anywhere, can remove

cc @alexanderleegs

Reply:
this wasn't actually used until this PR - this was used in `init`, which was never called till now. i decided to call it because it seemed to be setting transport defaults and cloudwatch log settings ._.